org-roam-dailies.el

```diff
@@ -327,25 +327,6 @@ negative, find note N days in the future."
 (define-key org-roam-dailies-map (kbd "v") #'org-roam-dailies-capture-date)
 (define-key org-roam-dailies-map (kbd ".") #'org-roam-dailies-find-directory)
 
-(define-obsolete-function-alias
```

Comment:
While I agree with `org-roam--directory-files-recursively` which is exactly what `compat` file are about, IMO obsoletisation of a function should stay in the module where function is defined.

Reply:
I kind of disagree, because it still promotes the usage of the obsolete functions and litters stuff. WDYT @jethrokuan?

Reply:
I'm ambivalent: I think having all obsolete declarations in a compat file helps with cleanup later on, but separates the function from its "alias". This is generally not an issue, it's a set-and-forget kind of thing. There are arguments for both sides, I guess it's just important to stick to one style.